packs/chef/actions/chef_install.py

```diff
@@ -20,9 +19,10 @@ class ChefInstaller(object):
         ('-d', '--download_path', {})
     ]
 
-    def install(self, install_method, options):
+    @staticmethod
+    def install(install_method, options):
         exit_code = 0
-        installer = getattr(ChefInstaller, install_method.capitalize())(options)
+        installer = getattr(sys.modules['lib'], install_method.capitalize())(options)
```

Comment:
Yeap. Both ways are nice, so I will change it the second one. Let's do the second way, interesting) I hope linting will be okay about such import.